recipes/WSJ0Mix/separation/README.md

```diff
@@ -61,10 +62,39 @@ Here are the SI - SNRi results (in dB) on the test set of WSJ0 - 2/3 Mix dataset
 | | SepFormer. WHAMR! |
 | --- | --- |
 |NoAugment | 11.4 |
+|SpeedAugment | 13.7|
```

Comment:
We should add here a link to the output folders stored in google drive.  I also think we should mention where are the pre-trained models and easy inference function in HuggingFace